core/src/main/scala/com/nvidia/spark/rapids/tool/tuning/AutoTuner.scala

```diff
@@ -1341,6 +1342,12 @@ trait AutoTunerConfigsProvider extends Logging {
     "spark.app.id"
   )
 
+  /**
+   * Default strategy for cluster shape recommendation.
+   * See [[com.nvidia.spark.rapids.tool.ClusterSizingStrategy]] for different strategies.
```

Comment:
does this cause any warning? I remember problems when I used the cross-reference in the doc.

Reply:
I did not see any warnings in the build console log.